assets/scss/components/_footer.scss

```diff
@@ -19,7 +19,9 @@
     @include media-breakpoint-down-md {
       max-width: 600px;
     }
-
+    .first-logo-link {
+      margin-right: 20px;
```

Comment:
I think you need to add some styling for this breakpoint or add additional padding under the chips logo so that they dont get mushed together at a small size. 
<img width="532" height="487" alt="Image" src="https://github.com/user-attachments/assets/8333a823-2da3-4d1b-a701-76bc6fe83b24" />